src/image/pixels.js

```diff
@@ -519,9 +519,9 @@ p5.prototype.get = function(x, y, w, h) {
  * }
  *
  * function setup() {
- *   image(img, 0, 0);
+ *   image(img, 0, 0, width, height);
  *   let d = pixelDensity();
- *   let halfImage = 4 * (img.width * d) * (img.height * d / 2);
+ *   let halfImage = 4 * (width * d) * (height * d / 2);
```

Comment:
I think it's good to leave img.width and img.height here as it's a little more clear to understand how this is half the image.